sty/omtext/omtext.dtx

```diff
@@ -329,7 +329,9 @@ use LaTeXML::Util::Pathname;
 \ifindex\makeindex\fi
 %</package>
 %<*ltxml>
+DeclareOption('showmeta',sub {PassOptions('metakeys','sty',ToString(Digest(T_CS('\CurrentOption')))); });
 DeclareOption('noindex','');
+ProcessOptions();
```

Comment:
Yes definitely, and. I think this is already added to .ltxml in the past, however somehow in .dtx it was not added as indeed the cross-maintaining also confuses me.